build/charts/theia/values.yaml

```diff
@@ -0,0 +1,27 @@
+clickhouse:
```

Comment:
It would make sense to have config options for the different theia images IMO (CH operator, Grafana, etc)

Reply:
Added